src/Lexer.hs

```diff
@@ -27,7 +29,8 @@ tokenizeLine :: String -> TokenLine
 tokenizeLine = (map tokenize) . words . removeComments
 
 removeComments :: String -> String
-removeComments = head . splitOneOf "--"
+--removeComments = head . splitOneOf "--"
+removeComments x = x ---- XXX
```

Comment:
why this change? this code was spliting the line by single comments and removing them